<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>888</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="addGiftList">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>891</width>
      <height>391</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="addKid">
     <attribute name="title">
      <string>Add Kid</string>
     </attribute>
     <widget class="QLineEdit" name="txtName_addKid">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>251</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>55</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Nombre:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>55</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Edad:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Se Porto Bien:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAddKid">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>10</y>
        <width>85</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="txtEdad">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>70</y>
        <width>46</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="txtSPB">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>120</y>
        <width>46</width>
        <height>27</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>46</width>
        <height>27</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>27</height>
       </size>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>80</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Longitud:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="txtLO">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>70</y>
        <width>46</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>120</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Latitud:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="txtLA">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>110</y>
        <width>46</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>481</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>0</y>
        <width>20</width>
        <height>181</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="addGift">
     <attribute name="title">
      <string>Add Gift List</string>
     </attribute>
     <widget class="QLineEdit" name="txtName_addGift">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>30</y>
        <width>311</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>40</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Nombre Kid:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Descripcion:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtDesc">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>331</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Control Paternal:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>70</y>
        <width>85</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Gift</string>
      </property>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>10</y>
        <width>20</width>
        <height>341</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAddListTo">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>60</y>
        <width>141</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Gift's List to</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listDesc">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>201</width>
        <height>241</height>
       </rect>
      </property>
     </widget>
     <widget class="QListWidget" name="listCP">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>110</y>
        <width>201</width>
        <height>241</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="txtCP">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>70</y>
        <width>78</width>
        <height>27</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>A [0 - 3]</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>B [4 - 8]</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>C [9 - 13]</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>D [+13]</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="search">
     <attribute name="title">
      <string>Search</string>
     </attribute>
     <widget class="QLineEdit" name="txtName_search">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>10</y>
        <width>171</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>55</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Nombre:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSearch">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>10</y>
        <width>85</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>search</string>
      </property>
     </widget>
     <widget class="Line" name="line_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>861</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="textInfo">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>331</width>
        <height>301</height>
       </rect>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="textListAntes">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>60</y>
        <width>241</width>
        <height>301</height>
       </rect>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="textListDespues">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>60</y>
        <width>241</width>
        <height>301</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>888</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
